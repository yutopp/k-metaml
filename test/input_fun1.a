fun fact (x) = if x=0 then 1 else x*fact (x - 1);
fact 4