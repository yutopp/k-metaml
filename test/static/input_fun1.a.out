Search results:

Solution 1:
V:Type -->
int
N:Int -->
0
